freedom-of-travel-web/src/main/java/nc/project/controller/AuthController.java

```diff
@@ -0,0 +1,53 @@
+package nc.project.controller;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import nc.project.auth.request.LoginRequest;
+import nc.project.auth.request.SignupRequest;
```

Comment:
Please remove the code which doesn't relate to the Login functionality

Reply:
done